doordeck-sdk/src/androidMain/kotlin/com/doordeck/multiplatform/sdk/model/responses/LockResponses.android.kt

```diff
@@ -0,0 +1,267 @@
+package com.doordeck.multiplatform.sdk.model.responses
+
+import com.doordeck.multiplatform.sdk.model.common.AuditEvent
+import com.doordeck.multiplatform.sdk.model.common.CapabilityStatus
+import com.doordeck.multiplatform.sdk.model.common.CapabilityType
+import com.doordeck.multiplatform.sdk.model.common.DayOfWeek
+import com.doordeck.multiplatform.sdk.model.common.UserRole
+
+data class LockResponse(
+    val id: String,
+    val name: String,
+    val colour: String? = null,
+    val start: String? = null,
+    val end: String? = null,
+    val role: UserRole,
+    val settings: LockSettingsResponse,
+    val state: LockStateResponse,
+    val favourite: Boolean,
+    val unlockTime: Double? = null
+)
+
+data class LockSettingsResponse(
+    val unlockTime: Double,
+    val permittedAddresses: List<String>,
+    val defaultName: String,
+    val usageRequirements: UsageRequirementsResponse? = null,
+    val unlockBetweenWindow: UnlockBetweenSettingResponse? = null,
+    val tiles: List<String>,
+    val hidden: Boolean,
+    val directAccessEndpoints: List<String> = emptyList(),
+    val capabilities: Map<CapabilityType, CapabilityStatus> = emptyMap()
+)
+
+data class UsageRequirementsResponse(
+    val time: List<TimeRequirementResponse>? = null,
+    val location: LocationRequirementResponse? = null
+)
+
+data class TimeRequirementResponse(
+    val start: String,
+    val end: String,
+    val timezone: String,
+    val days: List<DayOfWeek>
```

Comment:
Should we be using a Set for these?

Reply:
I updated it to use `Set<DayOfWeek>` in the common code and on all platforms except the `JVM`, which uses `EnumSet`.